SPECS/moby-compose/moby-compose.spec

```diff
@@ -0,0 +1,56 @@
+Summary:        Define and run multi-container applications with Docker
+Name:           moby-compose
+Version:        2.17.2
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Tools/Container
+URL:            https://github.com/docker/compose
+Source0:        https://github.com/docker/compose/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated GO dependencies from this tarball, since network is disabled during build time.
```

Comment:
Can you please create a "generate_source_tarball.sh" file so that 1) These manual steps are not necessary for the next guy, and 2) there is a chance the autopatcher/autoupgrader can assist with future updates?